src/Applications/LDAS_App/lenkf.j.template

```diff
@@ -68,6 +68,18 @@ else
    exit
 endif
 
+#######################################################
+#  if LADAS_COUPLING ==2, do atmens forcing processing 
+###################################################### 
+   if ( $LADAS_COUPLING == 2) then
```

Comment:
@saraqzhang : 
I'm slightly surprised this if block occurs at the very beginning of the job script.  Without seeing the python script, it's hard to tell which exact forcing data are ensemble-averaged.  
I assume that *all* of the forcing data that are needed for the 6-hour (?) interval are already staged before the LDAS simulation starts?   
Put differently, I'm wondering just how much of the current ADAS configuration we're hardwiring into the LDAS portion of the LADAS.

Reply:
@gmao-rreichle 
yes all of the forcing for current 6h segment are staged at enadas ( generated by the ensemble predictor segment in the last cycle of enadas).  they are available before the enadas process start the current cycle. 
 enadas does not produce ensmean for diag ( forcing included). so we do forcing averaging at ldas side when the run is coupled to enadas.    The enadas configuration such as stage dir path and ens sub dir structure are hardwired and limited  in the python code only.  Otherwise not hardwired but  specified at setup time (  ens size matching that of enadas, forcing path, forcing file tag etc.) .